python/CWE-094/CodeInjectionStatic.ql

```diff
@@ -20,7 +20,8 @@
 import python
 import semmle.python.Concepts
 import semmle.python.ApiGraphs
+private import semmle.python.security.dataflow.CodeInjectionCustomizations
 
 from DataFlow::Node sink
-where sink = any(CodeExecution e)
+where sink instanceof CodeInjection::Sink
```

Comment:
I don't think the changed line is semantically equivalent to the previous one. The current one selects the argument, as far as I can tell, as opposed to the previous one, which selects the call (I did not actually execute this query, just read the code, so I might be wrong). Is that intended?

Reply:
Still need to write the tests for this one